CHANGELOG.md

```diff
@@ -1,6 +1,51 @@
 # Changelog
 Notable changes to the framework should be documented here
 
+3.21.0:
+	- Add inventory for Raspberry Pi and DeviceTree devices (ENT-8628)
+	- Added check to make sure cf-execd is running after attempting self upgrade on Windows
```

Comment:
Bad title for https://github.com/cfengine/masterfiles/pull/2555, so here's a better changelog entry:

```suggestion
	- Added check to make sure cf-execd is running after attempting self upgrade on Windows
	- Added exception for settings.ldap.php file for ldap directory permissions (ENT-9697)
```

Reply:
@nickanderson ^^^